app/models/maker.rb

```diff
@@ -0,0 +1,54 @@
+require 'bcrypt'
+
+class Maker
+
+  include DataMapper::Resource
+  include BCrypt
+
+  has n, :peeps, :through => Resource
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)